pulsar-client/src/main/java/org/apache/pulsar/client/api/Message.java

```diff
@@ -92,6 +92,26 @@
      */
     long getEventTime();
 
+    /**
+     * Get the sequence id associated with this message. It is typically set by the applications via
+     * {@link MessageBuilder#setSequenceId(long)}.
+     *
+     * <p>If there isn't any sequence id associated with this event, it will return -1.
```

Comment:
There should be no condition in which the `sequenceId` or `producerName` are not set in the message

Reply:
remove this from javadoc